cmd/blueprints.go

```diff
@@ -173,6 +173,9 @@ type bpData struct {
 	// Metrics endpoint name
 	Management string
 
+	// Returns API documentation in Swagger syntax
```

Comment:
What logic do we want behind this

Reply:
It would be nice if we returned the api.json file we generated as the response for the explain end-point.  We would need to add it to the docker files in a known location.  That needs to be set by the environment so it still works in developer mode

I'd consider pulling this into its own issue and note handling as part of this one.